crates/cast/tests/cli/zk.rs

```diff
@@ -274,20 +283,26 @@ casttest!(test_zk_cast_mktx_contract_deploy, async |_prj, cmd| {
         .get_output()
         .stdout_lossy();
 
-    let output: String = output.trim().parse().unwrap();
-    assert!(output.contains("0x71f9042b80808402b275d08347731294000000000000000000000000000000000000800680b8849c4d535b000000000000000000000000000000000000000000000000000000000000000001000019f13e9e0a78abba31b36765bc5e4f708fdafc71b075be12759645331a0000000000000000000000000000000000000000000000000000000000000060000000000000000000000000000000000000000000000000000000000000000080a06a46c57e8b87d50d511995f36241e3a14550018796740905499f9cf843cfa074a077893132ce9905c876e7f40f59154139af47effcf566504be82664254b38eb4982010494bc989fde9e54cad2ab4392af6df60f04873a033a80f90323b903200000008003000039000000400030043f0000000100200190000000150000c13d000000000201001900000010002001980000002d0000613d000000000101043b000000e001100270000000110010009c000000200000613d000000120010009c0000002d0000c13d0000000001000416000000000001004b0000002d0000c13d000000000100041a000000ff0110018f000000800010043f0000001501000041000000370001042e0000000001000416000000000001004b0000002d0000c13d000000000200041a0000001601200197000000000010041b0000002001000039000001000010044300000120000004430000000f01000041000000370001042e0000000001000416000000000001004b0000002d0000c13d000000000100041a000000ff0210018f000000ff0020008c0000002f0000c13d0000001301000041000000000010043f0000001101000039000000040010043f000000140100004100000038000104300000000001000019000000380001043000000016021001970000000101100039000000ff0110018f000000000121019f000000000010041b0000000001000019000000370001042e0000003600000432000000370001042e0000003800010430000000000000000000000000000000000000000000000000000000020000000000000000000000000000004000000100000000000000000000000000000000000000000000000000fffffffc00000000000000000000000000000000000000000000000000000000000000000000000000000000d09de08a000000000000000000000000000000000000000000000000000000008381f58a4e487b710000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000240000000000000000000000000000000000000000000000000000000000000020000000800000000000000000ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff00000000000000000000000000000000000000000000000000000000000000000027b95d8697efbdb44a7508247e8c640a64fb3ead050f40cb23deb1910c5013158080"));
+    // Publish the contract deployment transaction
+    cmd.cast_fuse()
```

Comment:
I have a different approach in mind in https://github.com/matter-labs/foundry-zksync/pull/984 to test these functionalities without depending on anvil-zksync unnecessarily which pollutes the test scope. This is the same test framework that we previously introduced on anvil-zksync to mock rpc endpoints for writing well contained _unit tests_.